### PR TITLE
Fixing intermittent test failures - ANLYIS-169

### PR DESCRIPTION
**Pull request by ashensw**

### CODE REVIEW

File: product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java [hunk 1]
```diff
@@ -139,18 +139,33 @@ public void publishData() throws Exception {
...
-        long eventCount = analyticsDataAPI
-                .getRecordCount(MultitenantConstants.SUPER_TENANT_ID, "ORG_WSO2_IS_ANALYTICS_STREAM_LONGSESSIONS",
-                        Long.MIN_VALUE, Long.MAX_VALUE);
-        Assert.assertEquals(eventCount, 2, "========== Total long session count is invalid ==========");
+
+        final int EXPECTED_COUNT = 2;
+        final int MAX_WAIT_COUNT = 5;
+        final String LONGSESSIONS_TABLE = "ORG_WSO2_IS_ANALYTICS_STREAM_LONGSESSIONS";
+        int waitCount = 0;
+
```
Comment (isharak) [inline product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java]: remove  new line

File: product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java [hunk 2]
```diff
@@ -139,18 +139,33 @@ public void publishData() throws Exception {
...
+        while (analyticsDataAPI.getRecordCount(MultitenantConstants.SUPER_TENANT_ID, LONGSESSIONS_TABLE,
+                Long.MIN_VALUE, Long.MAX_VALUE) < EXPECTED_COUNT && waitCount < MAX_WAIT_COUNT) {
+
```
Comment (isharak) [inline product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java]: remove new line

File: product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java (hunk 1 above)
Comment (ashensw) [inline product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java]: Fixed with https://github.com/wso2/analytics-is/pull/288/commits/06102bc1ec294d9983507efcfe0b24212cebb984

File: product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java (hunk 2 above)
Comment (ashensw) [inline product/integration/tests-integration/tests/src/test/java/org/wso2/das4is/integration/tests/is/ISAlertLongSessionsAnalyticsTestCase.java]: Fixed with https://github.com/wso2/analytics-is/pull/288/commits/06102bc1ec294d9983507efcfe0b24212cebb984

**Pull request merged**